projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -633,16 +636,18 @@ export function checkQuoteInformationCard(isEditModeActive: boolean): void {
     'Verifies if the "Quote Information" card tile is in edit mode',
     checkQuoteInformationCard.name
   );
-  cy.get(cardBodySelector)
-    .contains(CARD_TITLE_QUOTE_INFORMATION)
-    .should('exist')
-    .then(() => {
-      if (isEditModeActive) {
-        cy.get(btnSelector).contains('Save').should('exist');
-      } else {
-        cy.get(btnSelector).contains('Save').should('not.exist');
-      }
-    });
+  cy.get(headerOverviewComponentSelector).within(() => {
```

Comment:
we should rename method from checkQuoteInformationCard to checkQuoteHeaderOverviewCardState